Update Protected Audience API page to acomodate prebid features (#56)
rtCamp/ps-analysis-tool-wiki: Protected-Audience.md

It will list out interest groups as you perform actions on specific elements, for example, if you click on e-commerce site and clicked/searched for a shoe, it will trigger an action that will add an interest group in your browser

<img alt="PSAT Protected Audience - Interest Groups" src="images/private-advertising/protected-audience/psat_v1.0.0_interest_groups_2025_06_30.png" width="1200" />

The tab shows that list of interest groups along with useful information such as event time, access type, name, owner, and expiration time. You can also filter based on the similar categories.

## Ad Units

### Prebid

The PSAT extension is an essential tool for publishers and ad operations professionals working with Prebid. It offers a transparent window into the complex world of header bidding, directly in your browser. When you visit a website with Prebid-enabled ads, the **Ad Units** tab populates with a wealth of actionable data.

<img src="images/private-advertising/protected-audience/prebid/psat_v1.0.0_ad_unit_prebid_2025_06_30.png" width="1200" alt="PSAT Prebid - Ad Units" />

> Here, you will instantly see all configured ad unit codes, their sizes, and the active bidders competing for them. For focused analysis, you can filter the view by a specific bidder.

Reviewer: ```suggestion
Here, you will instantly see all configured ad unit codes, their sizes, and the active bidders competing for them. For focused analysis, you can filter the view by specific bidders.
```

## Bids

### Prebid

A bid is a formal offer from an advertiser (a "bidder") to purchase a specific ad impression. These bids are the fundamental currency of the Prebid auction, allowing publishers to offer their inventory to multiple bidders simultaneously to maximize revenue.

<img src="images/private-advertising/protected-audience/prebid/psat_v1.0.0_bids_prebid_2025_06_30.png" width="1200" alt="PSAT Prebid - Bids" />

In the extension's Bids->Prebid tab, you can view detailed information for each bid, including the event time, bidder name, bid value (CPM), currency, ad unit code, and ad size.

You can also identify bidders who did not participate in the auction and filter the results for easier analysis.

### PAAPI

> In the Protected Audience API, bids are a crucial component of the ad auction process. bids in the Protected Audience API are the way advertisers express their interest in showing an ad to a user within a privacy-preserving, on-device auction environment.

Reviewer: ```suggestion
In the Protected Audience API, bids are a crucial component of the ad auction process. Bids in the Protected Audience API are the way advertisers express their interest in showing an ad to a user within a privacy-preserving, on-device auction environment.
```

<img alt="PSAT Protected Audience - Bids" src="images/private-advertising/protected-audience/paapi/psat_v1.0.0_bids_paapi_2025_05_30.png" width="1200" />

The Bids section contains two subsections, Received Bids and No Bids

The first section of Received Bids show list of bidders along with information such as event time, bid value, currency, Ad Unit, Ad Container Size, Media Type. You can also filter using the bidder using the same information.

> The second section Received Bids show list of bidders along with information of no bid.

Reviewer: ```suggestion
The second section, No Bids, shows a list of bidders who did not place a bid.
```

Author: Updated content as suggested.

<img alt="PSAT Protected Audience - No Bids" src="images/private-advertising/protected-audience/prebid/psat_v1.0.0_bids_nobids_2025_06_30.png" width="1200" />

### Timeline

<img src="images/private-advertising/protected-audience/psat_v1.0.0_bids_timeline_2025_06_30.png" width="1200" alt="PSAT Prebid Utilities - Timeline" />

> The Timeline is a diagnostic feature that provides a sequential, visual breakdown of events. For Prebid and PAAPI bids this would show the entire auction lifecycle, from request to render.

Reviewer: ```suggestion
The Timeline is a diagnostic feature that provides a sequential, visual breakdown of events. For Prebid and PAAPI bids, this shows the entire auction lifecycle, from request to render.
```

Author: Updated content as suggested.

<img src="images/private-advertising/protected-audience/prebid/utilities/psat_v1.0.0_prebid_utilities_config_2025_06_30.png" width="1200" alt="PSAT Prebid Utilities - Config" />

#### General Configuration (`pbjs.setConfig`)

This single, powerful function sets a wide range of auction options. Key configurations include:
-   **`priceGranularity`**: Defines the increments of bid prices (e.g., "0.10 increments"), which is crucial for how bids are passed to the ad server.

-   **`bidderTimeout`**: Sets the maximum time (in milliseconds) Prebid will wait for bidders to respond before closing the auction.

-   **`enableSendAllBids`**: By default, Prebid sends only the winning bid to the ad server. Setting this to `true` sends all bids, which is useful for analytics.

-   **`debug`**: When set to `true`, it enables verbose logging in the browser's console.

-   **`bidderSequence`**: By default, bid requests are sent asynchronously. This setting allows you to specify a `sequential` order, though this is rarely used as it can reduce the benefits of a parallel auction.

#### Bidder Settings (`pbjs.bidderSettings`)

> This object allows you to define rules that apply to all bidders or to a specific bidder, providing a powerful way to customize behavior without changing ad units.

Author: Updated content as suggested.

Prebid.js is modular. You can check which modules (e.g., bidder adapters, user ID modules) are currently installed using `pbjs.getConfig('modules')`. This is useful for debugging to ensure all necessary components are loaded.

### Events

<img src="images/private-advertising/protected-audience/prebid/utilities/psat_v1.0.0_prebid_utilities_events_2025_06_30.png" width="1200" alt="PSAT Prebid Utilities - Events" />

The Events Log is a dedicated tab that serves as a diagnostic tool by providing a clean, searchable, and sortable list of all warnings and errors that occur during a Prebid auction.

It features a prominent count of total issues for a quick health check and allows users to efficiently find and analyze specific problems to debug the auction process.

### Tools

<img src="images/private-advertising/protected-audience/prebid/utilities/psat_v1.0.0_prebid_utilities_tools_2025_06_30.png" width="1200" alt="PSAT Prebid Utilities - Tools" />

This tab offers centralized features that give you direct control over the auction on a live webpage. Key features often include:

-   **Shortcut to GAM Console**: Provides a direct link to the associated Google Ad Manager (GAM) console.

> -   **Override Tool**: A powerful testing utility that allows you to manually change various attributes of any auctions to simulate an test scenarios.

Reviewer: ```suggestion
-   **Override Tool**: A powerful testing utility that allows you to manually change various attributes of any auction to simulate test scenarios.
```

Author: Updated content as suggested.
